app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```
@@ -542,38 +542,37 @@ $chevron-indent-spacing: 7px;
 
 // Styles for top level navigation toggle button.
 .gem-c-layout-super-navigation-header__navigation-top-toggle-button {
-  @include govuk-font($size: 19, $weight: "bold", $line-height: 20px);
+  @include govuk-link-common;
+  @include govuk-link-style-no-visited-state;
+  @include govuk-link-style-no-underline;
+
+  font-size: 16px;
+  font-size: govuk-px-to-rem(16px);
```

Let's wrap this in a conditional to avoid legacy problems:

```suggestion
  @if $govuk-typography-use-rem {
    font-size: govuk-px-to-rem(16px);
  }
```

---

Or possibly a rebase against the main branch might sort this out.